index.html

```diff
@@ -54,7 +54,254 @@ <h3>LET’S CONNECT</h3>
                    </div>
                </section>
 
-               <div  id="about"></div>
+               <div  id="about" class="about">
```

Comment:
- [ ] Please I highly recommend wrapping the work section with a `section` tag.
```suggestion
               <section  id="about" class="about">
                 <div class='work-card'></div>
               </section>
```